404.php

```diff
@@ -11,8 +11,7 @@
 // in this file.
 
 $script_base = dirname($_SERVER["SCRIPT_NAME"]);
-$image_base_url = "https://" .
-	$_SERVER["SERVER_NAME"] . $script_base . "/images";
+$image_base_url = "/images";
```

Comment:
Is it *always* going to be `/`?  Or should we make it `/$script_base/images`?

If we stop supporting putting the web site in a non-root directory (which, at this point, is only useful for testing/developing the web site itself -- we no longer have to worry about mirrors running the Open MPI web site in a non-root folder), then the definition and use of `$script_base` in this file can go away.